<?xml version="1.0"?>
<launch>

    <!-- Numerically integrate twists into a pose -->
    <node name="vive" pkg="vive_tracking_ros" type="track_controllers.py" output="screen">
        <param name="config_file" value="$(find vive_tracking_ros)/config/vive_config.yaml" />
        <param name="publishing_rate" value="100" type="int" />
    </node>

    <node name="static_transform_publisher" pkg="tf" type="static_transform_publisher" args="0 0 0.0 0.0 0.0 0.0 world vive_world 1"/>

</launch>
